### PR TITLE
Info2-006-02-03: Add solution

### PR DESCRIPTION
**Pull request by paul891112**
add .c files

### CODE REVIEW

File: Informatics II/006/Task 2/Paul891112 Solution/task2.c [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#include<stdio.h>
+#include<stdlib.h>
+#define N 5
+
+
+void print(int *arr, int n);
+
+int* reverse(int *arr, int n);
+
+int* prepend(int *arr, int v);
```
Comment (C0DECYCLE) [inline Informatics II/006/Task 2/Paul891112 Solution/task2.c:10]: Redundant

File: Informatics II/006/Task 2/Paul891112 Solution/task2.c [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    return p;
+
+}
+
+
+int* prepend(int *arr, int v){
+    int* p = arr-1;
+    p[0] = v;
+    return p;
+}
```
Comment (C0DECYCLE) [inline Informatics II/006/Task 2/Paul891112 Solution/task2.c:37]: potentially problematic when "arr" is released later

File: Informatics II/006/Task 3/Paul891112 Solution/task3.c [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    int key;
+    struct node* next;
+};
+
+
+struct node* convertArraytoLinkedList(int *arr, int n);
+
+void print(struct node* curr);
+
+struct node* reverseLinkedList(struct node* head);
```
Comment (C0DECYCLE) [inline Informatics II/006/Task 3/Paul891112 Solution/task3.c:16]: Redundant

**Review by C0DECYCLE — CHANGES_REQUESTED** [review]
Minor changes else LGTM

**Review by Patertuck — CHANGES_REQUESTED** [review]
I think the point of task 2 was to use the pointers instead of arrays in the different functions. So not calling the accessing by arr[index] but rather *(arr + index). Correct me if I am wrong :)

**Review by C0DECYCLE — APPROVED** [review]

**Comment by C0DECYCLE** [comment]
My request LGTM👍 Before merge please also fix the point from @Patertuck

**Pull request merged**